apps/ws/src/.env.agent

```
@@ -0,0 +1,26 @@
+
+GLOBAL_CONTEXT_PATH=
+WS_CONTEXT_PATH=
+
+NEW_RELIC_ENABLED=false
+NEW_RELIC_APP_NAME="[LOCAL] - ws"
+NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED=false
+LOG_LEVEL=info
+
+SOCKET_IO_ADMIN_USERNAME=admin
+# "changeit" encrypted with bcrypt
+SOCKET_IO_ADMIN_PASSWORD_HASH=$2b$10$heqvAkYMez.Va6Et2uXInOnkCT6/uQj1brkrbyG3LpopDklcq7ZOS
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SOCKET_IO_ADMIN_PASSWORD_HASH key should go before the SOCKET_IO_ADMIN_USERNAME key

(UnorderedKey)